apps/jira-mapping/src/db/migrations/20201215160401-create-connections.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
```

Comment:
Let's add here a disclaimer, that using sqlite DB is just for demo purpose, in production tokens need to be stored in the encyprted protected storage such as Vault